lib/iex/lib/iex.ex

```diff
@@ -224,6 +224,16 @@ defmodule IEx do
 
   """
 
+  ## Exiting the shell
+
+  There are a few ways to quit the IEx shell:
```

Comment:
I think you have added this to the wrong place. :) It is in the middle of the code and not inside the documentation. :)